aiohttp/client.py

```diff
@@ -1399,6 +1399,7 @@ def request(
     read_bufsize: Optional[int] = None,
     max_line_size: int = 8190,
     max_field_size: int = 8190,
```

Comment:
Most of these options are defined in _RequestOptions, all the duplicates will need to be removed (as highlighted by the mypy error).